### DISCUSSION TITLE
安装完vimspectory插件后，无法显示VimspectorInstall命令？

### DISCUSSION THREAD

**Discussion by wxzvim** [nshen/learn-neovim-lua#98] [Q&A]
我在按照大佬的教程安装vimspectory插件后，也pip3更新了pynvim包，奈何就是VimspectorInstall命令出不来？
我重装插件没有效果，我系统是wsl ubuntu18的，，python环境是默认的python3.6.。。想问下，是什么原因？
怎么搞能出来VimspectorInstall命令，，，，快疯了

**Comment by nshen** [comment]
这个我也不太清楚。。好久没碰这个了，`:checkhealth` 看看可能是python版本的问题，各位大佬们有遇到的没有。

**Reply by wxzvim** [reply]
原因找到呢，wsl要安装python3.8的环境才行，checkhealth检查后，不支持python3.6环境
